src/formatting.tsx

```diff
@@ -98,6 +98,10 @@ export function formatDate (value?: string | null, dateFormat = DATE_FORMATS.dat
   return dateFnsFormat(value, dateFormat);
 }
 
+export function formatDateTime (value?: string) {
```

Comment:
This should, unless there's a good reason, be `value?: string | null` and should use `hasStringContent` at the top like `formatDate` does.